Source/relay/TourGuide/Sets/Sneaks.ash

```diff
@@ -0,0 +1,363 @@
+        return final;
+    }
+
+    SneakSource getStenchJellies() {
+        SneakSource final;
+
+        final.sourceName = "stench jelly";
+        final.url = "";
+        final.imageLookupName = "__familiar space jellyfish";
+
+        // You can get more than 3 in a run, but 3 is a good fairway estimate.
+        int likelyJellies = 3;
+
+        // Check if the familiar is usable & you have jellies in your inventory
+        boolean canUseJellyfish = $familiar[space jellyfish].familiar_is_usable();
+        int jellyCount = $item[stench jelly].available_amount();
+        int toastCount = $item[toast with stench jelly].available_amount();
+
+        // Check that you don't have full organs
+        int numberOfJelliesConsumable = min(stomachLeft + spleenRemaining, jellyCount + toastCount);
+
+        // Compare current extractions to the likelyJellies variable
+        int currentExtractions = get_property_int("_spaceJellyfishDrops");
+        int plausibleJellies = max(currentExtractions - likelyJellies, numberOfJelliesConsumable);
+
+        // This is an or because it's possible to pull a stench jelly
+        final.sneakCondition = canUseJellyfish || plausibleJellies > 0;
+
+        // Use plauisible jellies here, to make sure it's added if it's possible
+        final.sneakCount = plausibleJellies;
+        final.tileDescription = `<b>{plausibleJellies}x stench jellies</b> (have {clampi(toastCount,0,15)} on toast, {clampi(jellyCount,0,15)} as jelly)`;
+        return final;
+    }
+    
+    SneakSource getSpikos() {
+        SneakSource final;
+
+        final.sourceName = 'spikolodon spikes';
+        final.url = 'inventory.php?action=jparka';
+        final.imageLookupName = "__item jurassic parka";
+
+        int spikosLeft = clampi(5 - get_property_int("_spikolodonSpikeUses"), 0, 5);
+        
+        final.sneakCondition = lookupItem("jurassic parka").have();
+        final.sneakCount = spikosLeft;
+        final.tileDescription = `<b>{spikosLeft}x spikalodon spikes</b> left`;
+        return final;
+
+    }
+    
+    SneakSource getClaras() {
+        SneakSource final;
+
+        final.sourceName = `clara's bell`;
+        final.url = '';
+        final.imageLookupName = "__item clara's bell";
+
+        final.sneakCondition = $item[clara's bell].available_amount() > 0 && !get_property_boolean("_claraBellUsed");
+        final.sneakCount = get_property_boolean("_claraBellUsed") ? 0 : 1; 
+        final.tileDescription = `<b>{final.sneakCount}x clara's bell</b> charge left`;
+
+        return final;
+    }
+    
+    SneakSource getCinchos() {
+        SneakSource final;
+
+        final.sourceName = `fiesta exits`;
+        final.url = '';
+        final.imageLookupName = "__skill Cincho: Fiesta Exit";
+        final.sneakCondition = lookupItem("Cincho de Mayo").have();
+
+        // In the most recent draft of the PR (located here https://github.com/kolmafia/kolmafia/pull/1685 )
```

Comment:
Maybe we can clean up this comment now that it was merged. More importantly, do we need to bump the `since` version in relay-TourGuide.ash?

Reply:
yep, bumping since in my new commit.